lib/3_add_up_numbers.rb

```diff
@@ -5,5 +5,12 @@
 # Run `bin/check` from the command line to execute the automated tests.
 
 def add_up_numbers(arr)
-  # Write your code here.
+  
+  add_up_numbers = 0
+  arr.each_index do |i|
```

Comment:
Nice! While functional, using `each_index` feels a little indirect. Is there a method that loops through every element in an Array and uses the element itself as the block variable (the variable inside the `||`)?